X16 Reference - 09 - VERA Programmer's Reference.md

```diff
@@ -919,8 +921,9 @@ Thus the output frequency can be set in steps of about 0.373 Hz.
 
 *Example: to output a frequency of 440Hz (note A4) the **Frequency word** should be set to  440 / (48828.125 / (2^17)) = 1181*
 
-**Volume** controls the volume of the sound with a logarithmic curve; 0 is silent, 63 is the loudest.
-The **Left** and **Right** bits control to which output channels the sound should be output.
+**Volume** controls the volume of the sound with a logarithmic curve; 0 is silent, 63 ($3F)
+is the loudest. The **Left** and **Right** bits control to which output channels the sound
+should be output.
```

Comment:
I know you haven't changed this sentence but the first "output" (line 925) seems redundant.